dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/FunctionChoiceBehaviorConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Represents function choice behavior configuration produced by a <see cref="FunctionChoiceBehavior" />.
+/// </summary>
+public sealed class FunctionChoiceBehaviorConfiguration
+{
+    /// <summary>
+    /// Represents an AI model's decision-making strategy for calling functions.
+    /// </summary>
+    public FunctionChoice Choice { get; internal set; }
```

Comment:
Should this be init instead of internal set?  Aren't the AI libraries creating these, and don't they need to be able to set without having access to Abstractions internals?

Reply:
Yep, even though the `init` access modifier is stricter than the `set` one, it will still work. AI connectors will use the property to decide which function call behavior to use and map it to the underlying LLM model.